React/Modules/RCTServerUtils.m

```diff
@@ -0,0 +1,21 @@
+
+#import "RCTServerUtils.h"
+#import "RCTIPAddress.h"
+
+@implementation RCTServerUtils
+
++ (NSURL *)getServerURL:(NSString *)path
+{
+  NSUserDefaults *standardDefaults = [NSUserDefaults standardUserDefaults];
+  NSInteger port = [standardDefaults integerForKey:@"websocket-executor-port"] ?: 8081;
+
+#if TARGET_OS_SIMULATOR
```

Comment:
This is the setup I used in the F8 app https://github.com/fbsamples/f8app/blob/master/scripts/store-ip.sh, worked smoothly across dev/prod/simulator/device.